internal/format/testdata/markdown/table-Base.golden

```diff
@@ -60,6 +60,7 @@ followed by another line of text.
 | bar | baz | 4.5.6 |
 | foo | bar | 1.2.3 |
 | baz | baz | 4.5.6 |
+| foobar | git@github.com:module/path | v7.8.9 |
```

Comment:
One of the main problem mentioned at https://github.com/terraform-docs/terraform-docs/issues/427#issuecomment-797019807, is that this will become a "clickable email" address. The following for `makrdown` and `asciidoc` is preferred:

```suggestion
| foobar | [git@github.com:module/path](https://github.com/module/path) | v7.8.9 |
```

Reply:
Hi! yes, it's a bit hard to implement, but I'll try! tks!